slamd/materials/processing/strategies/aggregates_strategy.py

```diff
@@ -46,6 +46,7 @@ def create_model(cls, submitted_material):
         return Aggregates(
             name=submitted_material['material_name'],
             type=submitted_material['material_type'],
+            density=submitted_material.get('density', None),
```

Comment:
Wenn, wie in dem anderen Kommentar erwähnt, die Density nicht mehr optional ist, reicht hier einfaches indexing